react-native/react-native.d.ts

```diff
@@ -3985,7 +4124,7 @@ declare namespace  __React {
          * A RefreshControl component, used to provide pull-to-refresh
          * functionality for the ScrollView.
          */
-        refreshControl?: RefreshControl
+        refreshControl?: any
```

Comment:
Can you try `React.ReactElement<RefreshControlProperties>` ? If that does not work, fall back to `JSX.Element`